SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DynamicSubobjectsTest/DynamicSubobjectsTest.cpp

```diff
@@ -31,18 +36,31 @@
  *  - Cleanup:
  *    - Client 1 repossesses its default pawn.
  *    - The spawned Character is destroyed.
+ *
+ *
+ * A second test case is also tested with this same test.
+ * This tests that
+ * 1. The server adds a dynamic component to the actor
+ * 1. AReplicatedGASTestActor moves out of the client's interest
+ * 2. AReplicatedGASTestActor has the dynamic component removed
+ * 3. AReplicatedGASTestActor moves into the client's interest
+ * 4. The client sees AReplicatedGASTestActor no longer has the dynamic component
+ *
+ * This extra test case is implemented in steps 9.1 and 12.1
  */
 
-const static float StepTimeLimit = 10.0f;
+const static float StepTimeLimit = 15.0f;
 
 ADynamicSubobjectsTest::ADynamicSubobjectsTest()
 	: Super()
 {
-	Author = "Evi";
+	Author = "Evi&Arthur&Miron";
 	Description = TEXT("Test Dynamic Subobjects Duplication in Client");
 
 	CharacterSpawnLocation = FVector(0.0f, 120.0f, 40.0f);
 	CharacterRemoteLocation = FVector(20000.0f, 20000.0f, 40.0f); // Outside of the interest range of the client
+
+	TimeLimit = 100.0f;
```

Comment:
Does this test now work in native?

Reply:
yes

Reply:
we should make it run in native then - unfortunately the current map is manually created, and we would like it to be generated. Can discuss, could be a separate ticket.